nautobot/circuits/api/serializers.py

```diff
@@ -17,8 +17,11 @@
     TaggedObjectSerializer,
 )
 from nautobot.tenancy.api.nested_serializers import NestedTenantSerializer
-from .nested_serializers import *
+from .nested_serializers import NestedCircuitSerializer, NestedCircuitTypeSerializer, NestedProviderSerializer
 
+# Not all of these variable(s) are not actually used anywhere in this file, but required for the
+# automagically replacing a Serializer with its corresponding NestedSerializer.
+from .nested_serializers import NestedCircuitTerminationSerializer  # noqa: F401
```

Comment:
Please consolidate these imports.

Reply:
sorry, I thought I got them all, should be good now